lib/commands/new.ts

```diff
@@ -20,15 +21,20 @@ export class ${className} implements MigrationInterface {
 };
 
 export const newCommand = (opts: CommandNewOptions): string => {
-  const { migrationName, migrationsDir } = opts;
+  const { migrationName, migrationsDir, templateFile } = opts;
 
   if (!fs.existsSync(migrationsDir)) {
     fs.mkdirSync(migrationsDir);
   }
-
+  let template: string;
   const fileName = `${+new Date()}_${migrationName || 'Migration'}`;
   const className = `${migrationName || 'Migration'}${+new Date()}`;
-  const template = migrationTemplate(className);
+
+  if (templateFile && fs.existsSync(templateFile)) {
+    template = fs.readFileSync(templateFile).toString();
+  } else {
+    template = defaultMigrationTemplate(className);
+  }
```

Comment:
I'd would prefer to encapsulate this into a `getMigrationTemplate` function or similar and I think is better if instead of creating the migration with the default template when the template file don't exist that it return an error.